Week1/KOUOKAM DIDERIQUE/powerFunction.js

```diff
@@ -0,0 +1,8 @@
+
+/*This is a power function of a given two numbers say x to the power y
+can also be calculated using the Math library*/
+function power(x,y){
```

Comment:
Bill explained yesterday why this solution was wrong.
Mind trying out another attempt?